core/src/components/datetime/datetime.scss

```diff
@@ -73,11 +73,25 @@
   display: flex;
 }
 
+/**
+ * Safari 14 has an issue where Intersection
+ * Observer is incorrectly fired when
+ * unhiding the calendar content.
+ * To workaround this, we set the opacity
+ * of the content to 0 and hide it offscreen.
+ * TODO: This is fixed in Safari 15+, so remove
+ * when Safari 14 support is dropped.
+ */
 :host(.show-month-and-year) .calendar-next-prev,
 :host(.show-month-and-year) .calendar-days-of-week,
 :host(.show-month-and-year) .calendar-body,
 :host(.show-month-and-year) .datetime-time {
-  display: none;
+  @include position(null, null, null, -99999px);
```

Comment:
It's still possible to tab to the calendar body like this, so you have to hit tab several times to get from the month/year dropdown to the pickers. Does `visibility: hidden` still get around the intersection observer issue? Otherwise we might need to involve the JS too and add `tabindex="-1"` or something.

Reply:
Visibility works. Good catch!